src/main/java/sirius/kernel/nls/Formatter.java

```diff
@@ -393,6 +419,13 @@ private Block endBlock(List<Block> blocks, Block currentBlock, int index) {
         return currentBlock;
     }
 
+    private String escapeJS(String value) {
+        if (value == null) {
+            return value;
+        }
+        return value.replace("'", "\\'").replace("\"", "\\\"");
```

Comment:
Hmm, müsste man sich hier nicht auch noch drum kümmern, wenn schon ein Backslash im String ist, damit der nicht versehentlich was anderes escaped? Und wie ist es mit den Backticks von Template Strings? Und kann was schief gehen, wenn man das eine Zeichen innerhalb der anderen Quotes escaped?

[Das Internet](https://stackoverflow.com/questions/770523/escaping-strings-in-javascript) schlägt sowas vor:
```
JSON.stringify(sDemoString).slice(1, -1);
```